js/ui/grid_core/ui.grid_core.header_panel.js

```diff
@@ -48,9 +50,17 @@ const HeaderPanel = ColumnsView.inherit({
 
         this.executeAction('onToolbarPreparing', options);
 
-        if(options.toolbarOptions && !isDefined(options.toolbarOptions.visible)) {
+        if(options.toolbarOptions) {
+            if(!isDefined(options.toolbarOptions.visible)) {
+                options.toolbarOptions.visible = true;
+            }
```

Comment:
This `if` is needed cause in unittests `defaultOptions` isn't called.  Even if I set `initDefaultOptions: true` in `setupDataGridModules`, this value will be overriden cause in most tests options are set with `this.options = {...}`, not `this.option({...})`